crashes/index.html

```diff
@@ -61,6 +62,9 @@
 var gData = undefined;
 var gMap = {};
 
+// When populated, this is a map activity_date -> channel -> version
+var gLatestVersions = getLatestVersions();
```

Comment:
With limit=10 (at least, I *think* that is what is causing the problem), some options for "analyzed date" aren't showing (well, they show, but they don't update the table). Check the earliest date (2016-11-08) for an example of this. Gets a "TypeError: gMap[weekAgoDate] is undefined".